tests/boards/ehl_crb/gpio/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+#Copyright(c)2021 Intel Corporation
```

Comment:
I don't believe .yaml file needs copyright header...maybe that was the build error already mentioned?

Reply:
The build error has been resolved, I will remove the copyright header.